ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -357,7 +357,7 @@
         variable (as-> variable-json-str data
                        (concept-json->concept data)
                        (assoc data :originator-id user-id)
-                       (assoc data :native-id (string/lower-case (:name data))))]
```

Comment:
This change causes @ygliuvt's variable association tests to fail; without this, the variable ingest tests fail. Will ping @ygliuvt to see where casing needs to be lowered ...